### ISSUE TITLE
Package Request: SEGGER J-Link

### ISSUE THREAD

**Issue by Mistrical** [microsoft/winget-pkgs#27720]
# Package Requested

- [x] I would like someone else to build the manifest.
- [ ] I would like help so I can submit the manifest.
- [x] I have performed a search and couldn't find this package.
- [ ] I think there is a new version available and I have provided the URL.

---

## Please provide the following information if you can:

**Publisher**: SEGGER Microcontroller

**Publisher Url**: https://www.segger.com/

**Package Name**: SEGGER J-Link

**Package Url**: https://www.segger.com/downloads/jlink/

**Description**: Software for use with the J-Link debug probes

**Package Version**: V7.54b

**Installer URL**: https://www.segger.com/downloads/jlink/JLink_Windows_V754b_x86_64.exe

## For new versions:

**Current Package Identifier**: V7.54b

**Comment by KaranKad** [comment]
Download url leads to Terms of Use
![image](https://user-images.githubusercontent.com/71691514/133905611-a3a2f656-63bc-4a4a-b7e4-a3e11fe9c06a.png)

**Comment by Mistrical** [comment]
> Download url leads to Terms of Use

@KaranKad Does this mean it can't be integrated into winget?

**Comment by KaranKad** [comment]
Not right now, but it is possible to download files behind ELUA/TOS using [wget](https://www.gnu.org/software/wget/), winget could ask to install wget when downloading files behind ELUA/TOS.
![image](https://user-images.githubusercontent.com/71691514/134004020-64b1f1f1-1566-4d0d-b655-873de1476a13.png)

**Comment by vedantmgoyal2009** [comment]
@KaranKad Do you use oh-my-posh or something else for your terminal theme?

**Comment by KaranKad** [comment]
> @KaranKad Do you use oh-my-posh or something else for your terminal theme?

Yeah I use oh-my-posh and terminal themes from https://windowsterminalthemes.dev/

**Comment by Mistrical** [comment]
Could I have some help with this package? I believe that winget now supports .exe installers.

**Comment by starthal** [comment]
@KaranKad FWIW here is the equivalent `curl` arg: 

    --data 'accept_license_agreement=accepted&non_emb_ctr=confirmed&submit=Download+software'

(I'm not sure if the last two params are required)

I don't know how winget does its HTTP but it may be possible to include this POST data for a particular package.

**Comment by KaranKad** [comment]
Unfortunately, winget does not support custom arguments while downloading packages as of now.
